python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_condition.py

```diff
@@ -147,6 +147,40 @@ def parent_newer() -> "ParentNewerCondition":
 
         return ParentNewerCondition()
 
+    @staticmethod
+    def eager() -> "SchedulingCondition":
```

Comment:
I believe it does, but can we explicitly state that an `eager` scheduling condition will kick off if one of its parents is the target of an _ad hoc_ update?